### ISSUE TITLE
Jenkins: Add an option to not fail the job if no report is found

### ISSUE THREAD

**Issue by Kabhal** [gatling/gatling#1992]
We need a new option to "Gatling simulation tracking" command :
- Fail build if no reports : Yes/No

Currently, the job always fail if no report is found.
This would permit to create some generic Jenkins jobs, or even conditional Gatling execution, without failing the job.

**Comment by slandelle** [comment]
@Kabhal I'd like to know more about your use case, sorry but I still don't get it. Not being able to find the reports is something unexpected, and failing the build is then the best way to be notified.

> This would permit to create some generic Jenkins jobs

What do you mean exactly?

> or even conditional Gatling execution, without failing the job.

Looks like a maven profile or similar to me.

**Comment by Kabhal** [comment]
Perhaps my use case is too specific :) anyway I explain it :

As I've a lot of similar Java projects and jobs in Jenkins, I use the Template workflow plugin (https://wiki.jenkins-ci.org/display/JENKINS/Template+Workflows+Plugin) to generate them from some job templates (by the way, any other system using templates in Jenkins should lead to the same use case).

Gatling plugin is enabled by default in the template, but some projects do not have Gatling scenarios, so that, reports are not generated and Gatling plugin fails for those projects.
The workaround is to manually disable Gatling plugin for those projects each time I regenerate jobs from templates but it's painful.

To sum up :
"This would permit to create some generic Jenkins jobs" means that I could use generic Job templates with Gatling report analysis enabled by default, even if no report is created for a particular job.

**Comment by slandelle** [comment]
So if we were to add an option, you still have to:
* either manually enable Gatling on all Gatling project if it's disabled by default in your template
* or manually disable Gatling on all Gatling project if it's enabled by default in your template

It doesn't seem you would gain anything.

Or, you could have 2 templates, one with Gatling, and one without.

**Comment by Kabhal** [comment]
The templates will always have :
* Gatling : Disabled
* with option : "Fail build if no reports" : Disabled
So that, Gatling plugin will just silently ignore missing reports.
=> Nothing to do manually
